.bldr.toml

```diff
@@ -374,8 +374,6 @@ plan_path = "go14"
 plan_path = "go17"
 [gocd-server]
 plan_path = "gocd-server"
-[googlemock]
```

Comment:
Per [RFC 0007 ](https://github.com/habitat-sh/core-plans-rfcs/blob/master/_RFCs/0007-deprecating-packages.md) we should not remove this from builder.

Reply:
@predominant I thought we do.

> 4. Remove the plan from the `.bldr.toml` file in the core plans repo.
> 5. Disconnect the plan from Builder

Am I missing something?